src/zodiac.ts

```diff
@@ -0,0 +1,95 @@
+import type { Faker } from '.';
+
+/**
+ * Module to generate zodiac signs for birth dates.
+ */
+export class Zodiac {
+  constructor(private readonly faker: Faker) {
+    // Bind `this` so namespaced is working correctly
+    for (const name of Object.getOwnPropertyNames(Zodiac.prototype)) {
+      if (name === 'constructor' || typeof this[name] !== 'function') {
+        continue;
+      }
+      this[name] = this[name].bind(this);
+    }
+  }
+
+  /**
+   * Returns a random zodiac sign.
+   *
+   * @param birthdate The optional birthday of the person.
+   *
+   * @example
+   * faker.date.zodiac() // 'Pisces'
+   * faker.date.zodiac('01/01/1980') // 'Aquarius'
+   */
+  sign(birthdate?: string | Date): string {
+    // If birthday is null, return a random sign
+    if (!birthdate) {
+      return this.faker.random.arrayElement(this.faker.definitions.zodiac.sign);
+    } else {
+      // Birthday is a string, parse it as a date string
+      const date =
+        typeof birthdate === 'string' ? new Date(birthdate) : birthdate;
+
+      // Get the month and day of the month
+      const month = date.getMonth() + 1;
+      const day = date.getDate() + 1;
```

Comment:
Please check whether my change is correct:

```suggestion
      const day = date.getDate();
```

Code used for testing with (probably) expected values:

````ts
console.log("19/3", faker.zodiac.sign("0000-03-19T00:00:00Z")); // Pisces
console.log("20/3", faker.zodiac.sign("0000-03-20T00:00:00Z")); // Pisces
console.log("21/3", faker.zodiac.sign("0000-03-21T00:00:00Z")); // Aries

console.log("19/4", faker.zodiac.sign("0000-04-19T00:00:00Z")); // Aries
console.log("20/4", faker.zodiac.sign("0000-04-20T00:00:00Z")); // Aries
console.log("21/4", faker.zodiac.sign("0000-04-21T00:00:00Z")); // Taurus
````